### PR TITLE
Moved all geometry classes defined on the fly in Vizard to AnalyzerVisitors/GeometricInfo

### PR DESCRIPTION
**Pull request by ghugo83**
This is in the process of making Vizard disappear (to be continued ..).

### CODE REVIEW

File: include/AnalyzerVisitors/GeometricInfo.hh [hunk 1]
```diff
@@ -0,0 +1,185 @@
+#ifndef _GEOMETRICINFO_HH
+#define	_GEOMETRICINFO_HH
+
+#include <global_constants.hh>
+#include "VizardTools.hh"
+#include <Tracker.hh>
+#include <Analyzer.hh>
```
Comment (alkemyst) [inline include/AnalyzerVisitors/GeometricInfo.hh]: If the purpose is making Analyzer disappear, shouldn't we move the constants/variables/functions we need in Analyzer.hh into something different? I created AnalyzerTools for this, but any other choice is equally fine for me

Comment (alkemyst) [inline include/AnalyzerVisitors/GeometricInfo.hh]: More in general: do we need Analyzer.hh here at all?

**Comment by alkemyst** [comment]
Was this PR tested?

File: include/AnalyzerVisitors/GeometricInfo.hh (hunk 1 above)
Comment (ghugo83) [inline include/AnalyzerVisitors/GeometricInfo.hh]: Ok yes I have replaced #include <Analyzer.hh> by #include <SimParms.hh> #include <TagMaker.hh>

**Comment by ghugo83** [comment]
Yes, same output ;)

**Pull request merged**